updater/bin/run.sh

```diff
@@ -1,6 +1,11 @@
 #!/bin/bash
 set -e
 
+# WORKAROUND for https://github.com/ruby/resolv/issues/23, see also https://github.com/tinglesoftware/dependabot-azure-devops/pull/369
```

Comment:
```diff
- # WORKAROUND for https://github.com/ruby/resolv/issues/23, see also https://github.com/tinglesoftware/dependabot-azure-devops/pull/369
+ # This is a WORKAROUND for https://github.com/ruby/resolv/issues/23
+ # see also https://github.com/tinglesoftware/dependabot-azure-devops/pull/369
+ # see also https://github.com/tinglesoftware/dependabot-azure-devops/pull/834

Reply:
Sorry I missed the notification here from GitHub, will take care for this the next days!

Reply:
Okay. So this can be closed?

Reply:
if you mean merged yes, the issue is still open at `ruby/resolv`